File: SpeedTests/Java/count.java
public class Count {
    public static void main(String[] args) {
        long start = System.nanoTime();
        for (int i = 1; i <= 10000; i++) System.out.println(i);
        long elapsed = System.nanoTime() - start;
        System.out.printf("Elapsed: %.9f seconds\n", elapsed / 1e9);
    }
}

File: SpeedTests/C/count.c
// count.c
#include <stdio.h>
#include <time.h>

int main(void) {
    struct timespec start, end;
    clock_gettime(CLOCK_MONOTONIC, &start);

    for (int i = 1; i <= 10000; i++) {
        printf("%d\n", i);
    }

    clock_gettime(CLOCK_MONOTONIC, &end);
    double elapsed = (end.tv_sec - start.tv_sec) +
                     (end.tv_nsec - start.tv_nsec) / 1e9;
    printf("Elapsed: %.9f seconds\n", elapsed);
    return 0;
}
